plugins/woocommerce/src/Internal/FraudProtection/FraudProtectionController.php

```diff
@@ -58,10 +68,80 @@ public function on_init(): void {
 			return;
 		}
 
+		// Check Jetpack connection status and log if not connected.
+		$this->check_jetpack_connection();
+
 		// Future implementation: Register hooks and initialize components here.
 		// For now, this is a placeholder for the infrastructure.
 	}
 
+	/**
+	 * Check Jetpack connection status and log warning if not connected.
+	 *
+	 * This implements the fail-open pattern: the feature will continue to work
+	 * even if Jetpack is not connected, but API calls will fail gracefully and
+	 * allow all sessions through.
+	 *
+	 * @return void
+	 */
+	private function check_jetpack_connection(): void {
+		$connection_status = $this->connection_manager->get_connection_status();
+
+		if ( ! $connection_status['connected'] ) {
+			self::log(
```

Comment:
This runs on every `init`, which could lead to many warnings being logged in a noisy way (once per request). Should we figure out a way to throttle it?